netci.groovy

```diff
@@ -32,4 +32,60 @@ def branch = GithubBranchName
             Utilities.addGithubPushTrigger(newJob)
         }
     }
-}
\ No newline at end of file
+}
+
+// Add VSI job.
+// For now, trigger VSI jobs only on explicit request.
+def newVsiJobName = Utilities.getFullJobName(project, "vsi", false /* isPr */)
+
+def newVsiJob = job(newVsiJobName) {
+    description('')
+    label('windows-roslyn-internal-dev15-project-system')
+
+    // This opens the set of build steps that will be run.
+    steps {
+        // Build roslyn-project-system repo.
+        batchFile("build.cmd /release")
```

Comment:
FYI: Currently, build.cmd always runs unit tests after build. We may want to either separate this script into build.cmd and buildandtest.cmd like Roslyn OR add a new buildonly command script/argument to the existing script so we can avoid running unit tests unnecessarily.